packages/web-components/src/dapp-wallet-bridge/DappWalletBridge.js

```diff
@@ -188,11 +188,19 @@ export class DappWalletBridge extends LitElement {
   }
 
   render() {
-    const hasBridgeHref = this.bridgeHref !== null;
+    // Locate the bridge first.
+    if (this.bridgeHref === null) {
+      return html`<agoric-iframe-messenger
```

Comment:
@turadg The `<agoric-iframe-messenger>` doesn't seem to play well when you dynamically change its `src`. It only checks for a loading timeout on initial render, so collating these two as before resulted in losing a valuable error state if the bridge iframe never loaded.

Reply:
good to know. I wonder if we can make `agoric-iframe-messenger` able to handle that.

Reply:
I think we could fix it to but I'd like to keep this change contained to get it in quicker